### PR TITLE
Clean RSVP Template and Use new initialization for Common

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#65755](https://central.tri.be/issues/65755)

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | You must have a space after a closing parenthesis | [rsvp.php:54](https://github.com/moderntribe/event-tickets/blob/8633ccf4b3f8c3716aa506b84a80327c4054b9b5/src/views/tickets/rsvp.php#L54)
:x: | You must have a space after a closing parenthesis | [rsvp.php:57](https://github.com/moderntribe/event-tickets/blob/8633ccf4b3f8c3716aa506b84a80327c4054b9b5/src/views/tickets/rsvp.php#L57)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$ticket' | [rsvp.php:59](https://github.com/moderntribe/event-tickets/blob/8633ccf4b3f8c3716aa506b84a80327c4054b9b5/src/views/tickets/rsvp.php#L59)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$ticket' | [rsvp.php:59](https://github.com/moderntribe/event-tickets/blob/8633ccf4b3f8c3716aa506b84a80327c4054b9b5/src/views/tickets/rsvp.php#L59)
:x: | You must have a space after a closing parenthesis | [rsvp.php:86](https://github.com/moderntribe/event-tickets/blob/8633ccf4b3f8c3716aa506b84a80327c4054b9b5/src/views/tickets/rsvp.php#L86)
:x: | You must have a space after a closing parenthesis | [rsvp.php:138](https://github.com/moderntribe/event-tickets/blob/8633ccf4b3f8c3716aa506b84a80327c4054b9b5/src/views/tickets/rsvp.php#L138)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [rsvp.php:139](https://github.com/moderntribe/event-tickets/blob/8633ccf4b3f8c3716aa506b84a80327c4054b9b5/src/views/tickets/rsvp.php#L139)

> via jenkins/codesniffer-phpcs

File: src/views/tickets/rsvp.php [hunk 1]
```diff
@@ -41,120 +41,108 @@
...
-					</td>
-				</tr>
-				<?php
+			$is_there_any_product = true;
+			$is_there_any_product_to_sell = $ticket->is_in_stock();
+		?>
+		<tr>
+			<td class="tribe-ticket quantity" data-product-id="<?php echo esc_attr( $ticket->ID ); ?>">
+				<input type="hidden" name="product_id[]" value="<?php echo absint( $ticket->ID ); ?>">
+				<?php if ( $ticket->is_in_stock() ): ?>
```
Comment (borkweb) [inline src/views/tickets/rsvp.php]: Should you use `$is_there_any_product_to_sell`?

File: src/views/tickets/rsvp.php [hunk 2]
```diff
@@ -41,120 +41,108 @@
...
+			 * @var Event ID
+			 * @var Tribe__Tickets__Ticket_Object
+			 */
+			do_action( 'event_tickets_rsvp_after_ticket_row', tribe_events_get_ticket_event( $ticket->id ), $ticket );
+
+		}
+		?>
+
+		<?php if ( $is_there_any_product_to_sell ): ?>
+		<tr class="tribe-tickets-meta-row">
```
Comment (borkweb) [inline src/views/tickets/rsvp.php]: Everything within this `if` statement should be tabbed in by one more level.

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | You must have a space after a closing parenthesis | [rsvp.php:54](https://github.com/moderntribe/event-tickets/blob/8633ccf4b3f8c3716aa506b84a80327c4054b9b5/src/views/tickets/rsvp.php#L54)
:x: | You must have a space after a closing parenthesis | [rsvp.php:57](https://github.com/moderntribe/event-tickets/blob/8633ccf4b3f8c3716aa506b84a80327c4054b9b5/src/views/tickets/rsvp.php#L57)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$ticket' | [rsvp.php:59](https://github.com/moderntribe/event-tickets/blob/8633ccf4b3f8c3716aa506b84a80327c4054b9b5/src/views/tickets/rsvp.php#L59)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$ticket' | [rsvp.php:59](https://github.com/moderntribe/event-tickets/blob/8633ccf4b3f8c3716aa506b84a80327c4054b9b5/src/views/tickets/rsvp.php#L59)
:x: | You must have a space after a closing parenthesis | [rsvp.php:86](https://github.com/moderntribe/event-tickets/blob/8633ccf4b3f8c3716aa506b84a80327c4054b9b5/src/views/tickets/rsvp.php#L86)
:x: | You must have a space after a closing parenthesis | [rsvp.php:138](https://github.com/moderntribe/event-tickets/blob/8633ccf4b3f8c3716aa506b84a80327c4054b9b5/src/views/tickets/rsvp.php#L138)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [rsvp.php:139](https://github.com/moderntribe/event-tickets/blob/8633ccf4b3f8c3716aa506b84a80327c4054b9b5/src/views/tickets/rsvp.php#L139)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | You must have a space after a closing parenthesis | [rsvp.php:54](https://github.com/moderntribe/event-tickets/blob/7f09bfbc10c9fb6ea25a29fa8a84ffef94896637/src/views/tickets/rsvp.php#L54)
:x: | You must have a space after a closing parenthesis | [rsvp.php:138](https://github.com/moderntribe/event-tickets/blob/7f09bfbc10c9fb6ea25a29fa8a84ffef94896637/src/views/tickets/rsvp.php#L138)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [rsvp.php:139](https://github.com/moderntribe/event-tickets/blob/7f09bfbc10c9fb6ea25a29fa8a84ffef94896637/src/views/tickets/rsvp.php#L139)

> via jenkins/codesniffer-phpcs

**Comment by bordoni** [comment]
Ready for another 👁‍🗨

**Comment by borkweb** [comment]
👍

**Pull request merged**